### PR TITLE
 Junit5: Convert two ELK Stack test cases  ( testLogLevelSearch and testOperatorLogSearch )

### PR DESCRIPTION
**Pull request by hzhao-github**
Converted testLogLevelSearch() and testOperatorLogSearch() to use JUnit5

Jenkins job passed

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/1159
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/1178/

### CODE REVIEW

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLogging.java [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * To test ELK Stack used in Operator env, this Elasticsearch test does
+ * 1. Install Kibana/Elasticsearch 
+ * 2. Install and Start the operator with Elk Enabled
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLogging.java:74]: typo

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java [hunk 2]
```diff
@@ -93,6 +93,23 @@
   public static final String VOYAGER_CHART_NAME = "voyager";
   public static final String VOYAGER_CHART_VERSION = "12.0.0";
 
+  // ELK Stack and WebLogic logging exporter constants
+  public static final String ELASTICSEARCH_NAME = "elasticsearch";
+  public static final String ELASTICSEARCH_IMAGE = ELASTICSEARCH_NAME + ":6.8.0";
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java:98]: seems like an old version of ES to test?

Comment (hzhao-github) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java:98]: let me try the latest version

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLogging.java (hunk 1 above)
Comment (hzhao-github) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLogging.java:74]: changed Elk to ELK Stack and the operator to Operator, if you meant those

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java (hunk 2 above)
Comment (hzhao-github) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java]: Hi Mark, I have upgraded the ELK Stack(Elasticsearch, Kibana and Logstash) version to 7.8.1, latest version

**Pull request merged**